frontend/awx/resources/inventories/InventoryForm.cy.tsx

```diff
@@ -0,0 +1,344 @@
+import { CyHttpMessages } from 'cypress/types/net-stubbing';
+import { AwxItemsResponse } from '../../common/AwxItemsResponse';
+import { InstanceGroup } from '../../interfaces/InstanceGroup';
+import { Inventory } from '../../interfaces/Inventory';
+import { Label } from '../../interfaces/Label';
+import { Organization } from '../../interfaces/Organization';
+import { CreateInventory, EditInventory, InventoryCreate } from './InventoryForm';
+
+export type regularPayload = {
```

Comment:
Define please types with capital letters, its common practice and in that case it also helps us to differentiate between type and variable below with the same name.